src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -545,6 +545,23 @@ public void update(List<AbstractEndpoint> conferenceEndpoints, long bweBps)
                 synchronized (ssrcToSimulcastController)
                 {
                     ctrl = ssrcToSimulcastController.get(ssrc & 0xFFFF_FFFFL);
+
+                    if (ctrl != null)
+                    {
+                        if (trackBitrateAllocation.track != ctrl.getSource()) 
```

Comment:
It might be nice to add a comment about what this block is doing. You could also combine the two if statements into one